### ISSUE TITLE
Tensorflow internally creates operators

### ISSUE THREAD

**Issue by sj6077** [tensorflow/tensorflow#12592]
System information

Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
Yes, I made custom distributed inception code
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 14.04
TensorFlow installed from (source or binary): Unmodified source with RDMA Verbs enabled
TensorFlow version (use command below): 1.3.0-rc1
Python version: 2.7.12
Bazel version (if compiling from source): 0.5.1
CUDA/cuDNN version: 8.0/5.1.5
GPU model and memory: NVIDIA TITAN Xp PCIe 12GB (4 per node)

I tried to run distributed inception, and I removed some nodes by breaking control dependencies in the graph.  However, tensorflow creates operators such as "inception_v3/mixed_17x17x1280a/branch7x7x3/Conv_2/BatchNorm/AssignMovingAvg_1_S21149" internally and called it. Why this kind of operators are created and is there any way to prevent it?

**Comment by ali01** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
